server/common/utils/helpers.js

```diff
@@ -492,6 +492,23 @@ const formatItems = items => list =>
     };
   });
 
+const timestampFromDateString = date =>
+  Math.round(new Date(date).getTime() / 1000);
+
+const reportDataSortFunction = (a, b) => {
```

Comment:
Is the word 'Function' necessary?

Reply:
"Comparator" postfix would be better for this one.